base/array.jl

```diff
@@ -2366,6 +2366,7 @@ function findall(A::AbstractArray{Bool})
     end
     I
 end
+findall(A::AbstractVector{Bool}) = (1:length(A))[A]
```

Comment:
To make sure e.g. OffsetArrays don't break:
```suggestion
findall(A::AbstractVector{Bool}) = eachindex(A)[A]
```